src/Endpoints/class-base-api-proxy.php

```diff
@@ -72,17 +63,7 @@ abstract public function get_items( WP_REST_Request $request );
 	 * @return bool
 	 */
 	public function permission_callback(): bool {
```

Comment:
Question: Now that this is done, do we need the `permission_callback()` function, or can we just call `is_user_allowed_to_make_api_call()` directly?

Note: We can leave this as-is if we think it is clearer that way.

Reply:
As far as I know we cannot call `is_user_allowed_to_make_api_call()` directly because it is inside `$api` property ..... so seems like keeping the `permission_callback` is fine and clear.